### PR TITLE
Updated function in ObjCB.js

### PR DESCRIPTION
**Pull request by android-dep**
Adjusted the chatNotice function to be sendNotice, see the attached PDF regarding functions.
[Chaturbate.pdf](https://github.com/brandonxavier/CB-AppDevKit/files/100241/Chaturbate.pdf)

### CODE REVIEW

**Comment by agtg1966** [comment]
u have nothing i want

**Comment by malamel** [comment]
Yes

File: runtime/ObjCB.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
-/**
+﻿/**
+ * Update Date: 2/1/2016
+ * User: JHamerstix (jhamerstix@myself.com)
  * Created with JetBrains WebStorm
  * User: brandonxavier (brandonxavier421@gmail.com)
- * Date: 6/20/13
+ * Original Date: 6/20/13
```
Comment (malamel) [inline runtime/ObjCB.js:6]: Original

File: Docs/onLeave.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+gender: �m� (male), �f� (female), �s� (trans), or �c� (couple)
+Example Usage
+cb.onLeave(function(user) {
+    cb.chatNotice('Bye ' + user['user'] + '!');
+});
+Example Output
+Notice: Bye testuser!
+Notice: {u'user': u'testuser', u'in_fanclub': False,
+         u'has_tokens': False, u'is_mod': False,
+         u'gender': u'm', u'tipped_recently': True}
```
Comment (malamel) [inline Docs/onLeave.txt:22]: True

**Pull request closed without merging**